src/qt/rpcconsole.cpp

```diff
@@ -1078,7 +1063,6 @@ void RPCConsole::peerLayoutChanged()
         }
 
         // get fresh stats on the detail node.
-        stats = clientModel->getPeerTableModel()->getNodeStats(detailNodeRow);
```

Comment:
nit: not sure about the two lines above?

Reply:
Removed those.